docs/Contributing.asciidoc

```diff
@@ -431,6 +435,235 @@ Also find more details in
+* Database user: `geekotest`
+* Database container name: `postgres-openqa`
+
+==== Environment Variables
+
+Create a file that you can `source` whenever you want to work in this
+environment:
+
+File: `~/localopenqa.leap.sh`:
+[source,sh]
+----
+#!/bin/bash
+
+# The directory where openQA operates in and which creates everything:
+# * git repos
+# * openQA assets, pool directory, testresults
+# * configuration
+
+export OPENQA_BASEDIR=$HOME/openqadev.leap
+
+# The rest of the variables can likely stay like this
+
+# If you want to import database dumps from elsewhere
+#export SQL_DUMP_DIR=$HOME/sqldumps
+
+export OPENQA_CONFIG=$OPENQA_BASEDIR/config
+export OPENQA_REPOS="$OPENQA_BASEDIR/git"
+PATH=$OPENQA_REPOS/openQA/scripts:$PATH
+
+# ----------- CPAN ------------
+# Whenever necessary, you can install additional modules from CPAN like:
+# cpanm -l $OPENQA_BASEDIR/perl5 Some::Module@1.23
+export PERL5LIB=$OPENQA_BASEDIR/perl5/lib/perl5
+PATH=$OPENQA_BASEDIR/perl5/bin:$PATH
+
+export OPENQA_KEY=1234567890ABCDEF
+export OPENQA_SECRET=1234567890ABCDEF
+----
+
+==== Postgres Container
+
+To be able to keep the database when stopping the container, we create a
+named volume.
+We are using a Debian postgres container here, but it doesn't matter what
+you use.
+[source,sh]
+----
+### host ###
+podman volume create mypostgres
+podman run -it --rm --name postgres-openqa \
+    -v mypostgres:/var/lib/postgresql/data \
+    -p 5432:5432 \
+    -e POSTGRES_PASSWORD="x" \
+    -d \
+    docker.io/library/postgres:17
+
+# Optionally add this to mount a directory for sql dumps to import
+#    -v "$SQL_DUMP_DIR:/sqldumps" \
+----
+
+For the next step you need to execute commands in the container:
+
+[source,sh]
+----
+# Create the role and database
+podman exec -it postgres-openqa su - postgres
+
+createuser --no-createdb --pwprompt geekotest
+# enter password ('x' in this example)
+createdb -O geekotest openqa-local
+----
+
+==== Distrobox
+
+You can use a leap container and install the requirements, but you can also
+build your own container with the requirements already installed.
+
+[source,sh]
+----
+### host ###
+distrobox create -i registry.opensuse.org/opensuse/leap:15.6 -n box-openqadev
+distrobox enter box-openqadev
+
+### distrobox ###
+
+# Leap 15.6
+zypper addrepo -G -p 95 https://download.opensuse.org/repositories/devel:/openQA/15.6 devel-openqa \
+    && zypper addrepo -G -p 95 https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.6/15.6/ devel-openqa-leap \
+
+# Tumbleweed
+#zypper addrepo -G p 95 https://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed devel-openqa \
+#        && zypper addrepo -G -p 95 https://download.opensuse.org/repositories/SUSE:/CA/openSUSE_Tumbleweed/SUSE:CA.repo \
+
+zypper -n ref && zypper -n install ca-certificates-suse \
+        os-autoinst-devel openQA-devel \
+        os-autoinst-scripts-deps \
+        os-autoinst-scripts-deps-devel \
+        openssh-clients \
```

Comment:
openssh-clients is necessary to do `git clone git@...`

Should we add it to os-autoinst and openQA requirements?

Reply:
The question is, should it be only a development requirement?
Even in normal instances people would want to clone via ssh. Or should it be just a `Recommends`?

Reply:
the dependency was introduced with https://github.com/os-autoinst/openQA/pull/6611 so I guess you can remove this line